Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+				ON		l.request_session_id = c.session_id
+				OUTER APPLY sys.dm_exec_sql_text(c.most_recent_sql_handle) t
+				WHERE	l.resource_database_id = DB_ID()
+				AND     l.request_status = 'GRANT'
+				AND		l.resource_type = 'APPLICATION'
+				AND     l.resource_description like '%[[]' + @_shortLockName + ']%'
+				AND     a.total_elapsed_time > @_longRunningThresholdMilliSeconds            
+				ORDER BY a.total_elapsed_time DESC
+			END
+
+			IF(@_statusMessage <> '')
+			BEGIN
+				RAISERROR('%s', 0, 231, @_statusMessage) WITH NOWAIT
+
+				SET @_statusMessage = 'Servicing lock attempt ' + CONVERT(VARCHAR(10), @_attempt) + ' result: Long running request detected.'
+				RAISERROR (@_statusMessage, 0, 231) WITH NOWAIT
+			END
+			ELSE 
+			BEGIN
+				SET @_acquireLockStart = GETUTCDATE()
+        
+				SET @_statusMessage = 'Acquiring servicing lock: @attempt=' + CONVERT(VARCHAR(10), @_attempt) + '@timestamp=' + CONVERT(VARCHAR, GETUTCDATE(), 109)
+				RAISERROR (@_statusMessage, 0, 231) WITH NOWAIT
+
+				if @_attempt = @_acquireLockMaxAttempts - @_lastAttemptsToKillSessions + 1
+					SET @_acquireLockMilliseconds = @_acquireLockMilliseconds + 10 * 1000
+
+				EXEC @_result = sp_getapplock  @Resource = @_acquireLock, @LockMode = 'Exclusive', @LockTimeout = @_acquireLockMilliseconds
+       
+				if @_result >= 0
+					BREAK
+
+				SET @_statusMessage = 'Servicing lock attempt ' + CONVERT(VARCHAR(10), @_attempt) + ' result: Timed out while acquiring the lock.'
+				RAISERROR (@_statusMessage, 0, 231) WITH NOWAIT
+			END
+
+			SET @_attempt = @_attempt + 1
+
+			SET @_noWaitAttempt = 1
+			WHILE @_noWaitAttempt <= 15
+			BEGIN
+				WAITFOR DELAY '00:00:01';
+				EXEC @_result = sp_getapplock  @Resource = @_acquireLock, @LockMode = 'Exclusive', @LockTimeout = 0
+				IF @_result >= 0
+					BREAK
+
+				SET @_noWaitAttempt = @_noWaitAttempt + 1
+			END
+
+			IF @_result >= 0
+				BREAK
+		END
+	END
+
+    IF @_result < 0
+    BEGIN
+        ROLLBACK
+		DROP VIEW _vw_tmpSqlFilesTable
+		DROP TABLE _tmpSqlFilesTable
+		PRINT 'Failed to acquire exclusive lock. Dropping temporary table, view and rolling back.'
+        RAISERROR('%%error="800070";%%:Failed to acquire an exclusive servicing lock', 16, -1)
+        RETURN
+    END
+
+    SET @_statusMessage = 'Lock acquired:' + CONVERT(VARCHAR, GETUTCDATE(), 109) + ' time to acquire lock:' + CONVERT(VARCHAR, DateDiff(ms, @_acquireLockAttemptStart, GETUTCDATE()), 109) + ' ms'
+    RAISERROR (@_statusMessage, 0, 231) WITH NOWAIT
+END
+PRINT 'Complete acquiring applock if specified'
+
+PRINT 'Begin executing scripts'
+
+DECLARE batchCursor CURSOR LOCAL FAST_FORWARD FOR
+    SELECT  SqlScriptContent
+    FROM    _tmpSqlFilesTable
+	ORDER BY ScriptIndex ASC
+
+OPEN batchCursor
+
+FETCH   NEXT FROM batchCursor
+INTO    @_batch
+
+WHILE @@FETCH_STATUS = 0
+BEGIN
+    SET @_batchStartTime = GETUTCDATE()
+    SET @_batchMessage = 'Executing Script: ' + CONVERT(VARCHAR(25), @_batchStartTime, 21)
```

Comment:
>Executing Script:  [](start = 26, length = 18)

it suggests that whatever follows ':' is script content for script file name.
Can we change this to "<time>: Executing Script <content>